docs/runbooks/CnaoNotReady.md

```diff
@@ -0,0 +1,30 @@
+# CnaoNotReady
+
+## Meaning
+
+Cluster-network-addons-operator (CNAO) deploys additional networking components on top of the cluster.
+This alert fires when CNAO CR is not ready, which means that one of the deployed components is not ready.
```

Comment:
Let's add the name of the CR kind NetworkAddonsConfig

Reply:
https://github.com/kubevirt/monitoring/pull/21